recipes/aws-c-auth/all/conanfile.py

```diff
@@ -62,7 +62,7 @@ def requirements(self):
             self.requires("aws-c-common/0.9.6", transitive_headers=True, transitive_libs=True)
             self.requires("aws-c-cal/0.6.9")
             self.requires("aws-c-io/0.13.35", transitive_headers=True, transitive_libs=True)
-            self.requires("aws-c-http/0.7.14", transitive_headers=True)
+            self.requires("aws-c-http/0.8.1", transitive_headers=True)
```

Comment:
Hi! Thanks a lot for the contribution @toge 
Isn't this going to conflict with the other recipes that require `aws-c-http/0.7.14` ?
Is this bump needed for the new version? I can see that are other recipes that depend on `0.7.14`:

```
./recipes/aws-c-mqtt/all/conanfile.py:            self.requires("aws-c-http/0.7.14")
./recipes/aws-crt-cpp/all/conanfile.py:            self.requires("aws-c-http/0.7.14", transitive_headers=True)
./recipes/aws-c-s3/all/conanfile.py:            self.requires("aws-c-http/0.7.14")
```

Aren't they used together?

Reply:
@czoido
aws-c-http/0.7.14 is required to upgrade aws-c-s3.
And awsc-http is also used by aws-c-auth, which is used by aws-c-s3, so an update is required.

aws-crt-cpp is used by aws-c-s3 and aws-c-mqtt, aws-c-http, and aws-c-auth.
Therefore, when considering aws-crt-cpp, the aws-c-http version needs to be increased for all aws-c-s3, aws-c-mqtt, and aws-c-auth.

![image](https://github.com/conan-io/conan-center-index/assets/465629/19959ab0-10c8-4fe2-8c8e-41226aa08dae)